pkg/build/api/v1/types.go

```diff
@@ -402,6 +408,10 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// ExternalBuildStrategy defines input parameters specific to an External build.
+type ExternalBuildStrategy struct {
```

Comment:
the buildconfig should have the url to the pipeline engine.

the build, once created, should have the url to the pipeline instance (job instance).

the buildconfig should not be updated w/ specific information about the running pipeline, that should be represented in the build object.